genre/genre.rb

```diff
@@ -0,0 +1,15 @@
+class Genre
+  attr_accessor :name
+  attr_reader :id, :items
+
+  def initialize(name, id = Random.rand(1..1000))
+    @name = name
+    @id = id
+    @items = []
+  end
+
+  def add_item(item)
+    item.genre = self
```

Comment:
- [ ] As suggested by the previous reviewer, kindly fix this error 🐛 that occurs when trying to create a genre  👇
 
![image](https://user-images.githubusercontent.com/46839548/196927627-339f596f-3186-4585-b123-9226070025d7.png)